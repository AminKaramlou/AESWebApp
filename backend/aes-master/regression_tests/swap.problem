2;
A: 1
B: 1
C: 0.5
;
;
