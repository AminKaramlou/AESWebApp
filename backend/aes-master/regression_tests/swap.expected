Schedule is feasible
 -  All jobs are allocated to exactly one machine
Schedule does satisfies user fixed decisions
 -  All jobs satisfy all fixed decisions
Schedule is not efficient because:
 -  Job A can be allocated from machine 1 to 2 to reduce by 0.5
 -  Jobs A and C can be swapped with machines 1 and 2 to reduce by 0.5
 -  Job B can be allocated from machine 1 to 2 to reduce by 0.5
 -  Jobs B and C can be swapped with machines 1 and 2 to reduce by 0.5
